packages/eslint-config-khan/index.js

```diff
@@ -125,10 +125,37 @@ module.exports = {
         /**
          * typescript rules
          */
-        "@typescript-eslint/no-explicit-any": "off",
+
+        // Disallow the `any` type.
+        //
+        // We've disabled this rule because most repos have too many uses of `any`
+        // for this to be useful.  Individual repos can enable this rule if they're
+        // in a position to do so.
+        //
+        // Docs: https://typescript-eslint.io/rules/no-explicit-any
+        "@typescript-eslint/no-explicit-any": OFF,
```

Comment:
Even if the rule is self explanatory, I think that it's useful to have a rationale as to why we're enabling/disabling it.